datalad/crawler/nodes/annex.py

```diff
@@ -988,11 +999,12 @@ def finalize(self, tag=False, existing_tag=None, cleanup=False):
           Also `last_version` is provided as the last one from stats.versions (None
           if empty)
         existing_tag: None or '+suffix', optional
-          What to do if tag already exists, if None -- warning is issued. If `+suffix`
+          What to do if tag already exists, if None:\, warning is issued. If `+suffix`,
```

Comment:
Nope, this was me trying to delete `:` but pressing the backslash and still managing to add the comma! It'll be fixed.